scripts/vendor.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+if [[ -z "$VENDOR_MODULE" ]]; then
+	>&2 echo "Invalid or missing hugo module: \$VENDOR_MODULE)"
+	exit 1 
+fi
+
+output=$(mktemp -d -t hugo-vendor-output.XXXXXXXXXX)
+
+docker buildx bake vendor \
+	"--set=*.output=type=local,dest=${output}"
+rm -r _vendor
+cp -R $output/* .
+rm -rf "$output"
```

Comment:
nit: could use a trap for cleaning up temp folder

Reply:
done